pyproject.toml

```diff
@@ -54,6 +54,8 @@ dev = [
     "stac-geoparquet[pgstac]",
     "types-python-dateutil",
     "types-requests",
+    "pytest-recording>=0.13.2",
+    "urllib3<2; python_version < '3.10'",
```

Comment:
There's a known issue (https://github.com/kevin1024/vcrpy/issues/688) with Python 3.9 and earlier that requires us to use and older **urlllib3**.